src/strategies/snet-stakers/index.ts

```diff
@@ -0,0 +1,67 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'bonustrack';
+export const version = '0.1.1';
+
+type StakeInfoResponse = {
+  found: boolean;
+  approvedAmount: BigNumber;
+  pendingForApprovalAmount: BigNumber;
+  rewardComputeIndex: BigNumber;
+  claimableAmount: BigNumber;
+};
+
+type StakeResult = Record<string, StakeInfoResponse>;
+
+const stakingAbi = [
+  'function getStakeInfo(uint256 stakeMapIndex, address staker) public view returns (bool found, uint256 approvedAmount, uint256 pendingForApprovalAmount, uint256 rewardComputeIndex, uint256 claimableAmount)'
+];
+
+const parseNumber = (value: BigNumberish): BigNumber => BigNumber.from(value);
+
+const parseStakeInfo = (value: StakeInfoResponse) => ({
+  approvedAmount: parseNumber(value.approvedAmount),
+  pendingApprovalAmount: parseNumber(value.pendingForApprovalAmount)
+});
+
+const computeStakeBalance = (value: StakeInfoResponse) => {
+  const { approvedAmount, pendingApprovalAmount } = parseStakeInfo(value);
+  return approvedAmount.add(pendingApprovalAmount);
+};
+
+const multiCallerFactory = (network, provider, blockTag) => (abi) =>
+  new Multicaller(network, provider, abi, { blockTag });
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = 'latest';
```

Comment:
Same here https://github.com/snapshot-labs/snapshot-strategies/pull/202#discussion_r758960400

Reply:
fixed in https://github.com/snapshot-labs/snapshot-strategies/pull/203/commits/732e6e477b32e5918ba826bd1604f3ae6c244237